scala/ppml/demo/start-fl-client.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+export ENCLAVE_KEY_PATH=YOUR_LOCAL_ENCLAVE_KEY_PATH
+export DATA_PATH=YOUR_LOCAL_DATA_PATH
+export KEYS_PATH=YOUR_LOCAL_KEYS_PATH
+export DOCKER_IMAGE=intelanalytics/bigdl-ppml-trusted-fl-graphene:2.1.0-SNAPSHOT
+
+arg=$1
+case "$arg" in
```

Comment:
split hfl1 into hfl and 1. In that case, customer can change that number. For example 3